Editor/SoundSourceEditor.cs

```diff
@@ -18,11 +19,29 @@ public override void OnInspectorGUI()
             SerializedProperty src = serializedObject.FindProperty("Src");
             SerializedProperty volume = serializedObject.FindProperty("Volume");
             SerializedProperty loop = serializedObject.FindProperty("Loop");
+            SerializedProperty distance = serializedObject.FindProperty("Distance");
 
             EditorGUILayout.PropertyField(id);
             EditorGUILayout.PropertyField(src);
             EditorGUILayout.PropertyField(volume);
             EditorGUILayout.PropertyField(loop);
+            
+            EditorGUI.BeginChangeCheck();
+            EditorGUILayout.PropertyField(distance);
+            if (EditorGUI.EndChangeCheck())
+            {
+                if (!(serializedObject.targetObject is SoundSource soundSource))
+                {
+                    return;
```

Comment:
This might cause changes not being saved. See line 46.

Reply:
Removed checking `TargetObject`.